cmip6_downscaling/workflows/utils.py

```diff
@@ -16,8 +15,10 @@
 
 import cmip6_downscaling.config.config as config
 
-intermediate_cache_path = config.return_azure_config()["intermediate_cache_path"]
-connection_string = config.return_azure_config()["serializer"]
+cfg = config.CloudConfig()
+
+intermediate_cache_path = cfg.intermediate_cache_path
```

Comment:
how does the variable assignment for `intermediate_cache_path` differ from the `intermediate_cache_store` assignment in the flow itself?

Reply:
Good catch! They are identical.